001-cron.php

```diff
@@ -56,4 +56,14 @@ public function block_admin_page() {
 	}
 }
 
-new WPCOM_VIP_Cron_Control;
+// Allow testing of new approach to cron execution
+$whitelisted_sites = array();
+if ( VIP_GO_ENV && in_array( FILES_CLIENT_SITE_ID, $whitelisted_sites ) ) {
+	add_filter( 'wpcom_vip_go_enable_new_cron_control', '__return_true' );
```

Comment:
Do we need the filter here? We could just do the `require` here and add an `else` clause as a fallback.